moped-editor/src/components/GridTable/SearchBar.js

```diff
@@ -56,6 +56,28 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Create text to show advanced filters and logic applied in the UI
+ * @param {Object} filters - The current filters applied
+ * @param {Boolean} isOr - The current logic applied
+ * @returns {string} - The text to display
+ */
+const makeFilteredByText = (filters, isOr) => {
+  const filtersCount = Object.keys(filters).length;
+  let filteredByText = "Filtered by ";
+
+  /* Only show logic string if more than one filter applied */
+  if (filtersCount === 1) return filteredByText;
+
+  if (isOr) {
+    filteredByText += "any ";
+  } else {
+    filteredByText += "all ";
+  }
+
+  return filteredByText;
+};
+
```

Comment:
this is great. and i am inspired to start thinking about how we can improve some of the filter UX ✨